src/mockModules/mock-user/mock-user.controller.ts

```diff
@@ -0,0 +1,111 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpStatus,
+  Param,
+  ParseUUIDPipe,
+  Patch,
+  Post,
+  Res,
+} from "@nestjs/common";
+import { ApiOperation, ApiResponse, ApiTags } from "@nestjs/swagger";
+import { CreateMockUserDto } from "./dto/create-mock-user.dto";
+import { ResponseMockUserDto } from "./dto/response-mock-user.dto";
+import { UpdateMockUserDto } from "./dto/update-mock-user.dto";
+import { MockUserService } from "./mock-user.service";
+
+@Controller("user")
+@ApiTags("mockFracService/user")
+export class MockUserController {
+  constructor(private readonly mockUserService: MockUserService) {}
+
+  @Post()
+  @ApiOperation({ summary: "create new mock user" })
+  @ApiResponse({ status: HttpStatus.CREATED, type: ResponseMockUserDto })
+  async create(@Body() createMockUserDto: CreateMockUserDto, @Res() res) {
+    try {
+      const user = await this.mockUserService.create(createMockUserDto);
+      return res
+        .status(HttpStatus.OK)
+        .json({ data: user, message: "Successfully done" });
+    } catch (error) {
+      return res
```

Comment:
use getPrismaErrorStatusAndMessage to get error message and status for all controllers

Reply:
Done